### PR TITLE
Initial debugger documentation

### PR DESCRIPTION
**Pull request by zherczeg**

### CODE REVIEW

**Comment by zherczeg** [comment]
Let me know if you need more information.

File: docs/Debugger.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The debugger protocol uses packets, although the connection layer can split
+these packets into smaller ones if needed. The maximum packet length is defined by
+the connection layer. For example, it is 125 for the websockets layer. When a
+message is longer than the maximum packet size, it is split into a sequence of
+packets. The type of the last packet has an `_END` prefix. The following
+example shows the transmission of strings:
+
+Characters of a string can be 8 or 16 bit long. When a string is transmitted,
+it is split into a sequence a packets, end the type of these packets is either
```
Comment (clover2123) [inline docs/Debugger.md:14]: maybe typo at this line?
> it is split into a sequence `of` packets, `and` the type of these packets is either

Other looks good :)

**Review by ksh8281 — APPROVED** [review]

**Comment by zherczeg** [comment]
Thank you for the review. Besides the requested change, I have added some words about enabled/disabled breakpoint instructions.

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**